### PR TITLE
Creating secrets

### PR DESCRIPTION
**Pull request by missmesss**
Copied the general procedure from AppStudio docs.
Added a new section "Creating secrets for apps sourced from GitLab".

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc [hunk 1]
```diff
@@ -1 +1,51 @@
...
+* Each workspace can contain only one set of credentials for each Source Control Management (SCM) system, such as GitHub or GitLab. It means you can’t add two different tokens for a GitLab repository to a single workspace.
+
+* If you upload a GitLab access token to a workspace, {ProductName} won’t use the global GitHub application when accessing GitHub repositories.
+====
+
+.Additional resources
+
+* For more information about GitLab access tokens, see link:https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html[Personal access tokens].
+
+* To configure push secrets for your Build and Release pipelines, see link:https://github.com/konflux-ci/konflux-ci?tab=readme-ov-file#configuring-a-push-secret-for-the-build-pipeline[Configuring push secrets] in the Konflux GitHub repository.
```
Comment (missmesss) [inline docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc:51]: Check we want to link to a readme.

File: docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc [hunk 2]
```diff
@@ -1 +1,81 @@
...
+. For **Upload the file with value for your key or paste its contents**, do one of the following:
+    * Click **Upload** to browse to, select, and upload the file that contains your key value.
+    * Drag the file that contains your key value into the space under **Upload**.
+    * Paste the contents of the file that contains your key value into the space under **Upload**.
+  Click **Clear** to remove the contents of the space under **Upload**.
+. Optional: Click **Add another key/value**.
+. Optional: Under **Labels**, add a label to tag or provide more context for your secret.
+. Click **Add secret**.
+
+== Creating secrets for GitLab-sourced apps
```
Comment (missmesss) [inline docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc:28]: @mshaposhnik Hi, can you please review a procedure that includes your content? It starts on line 28, "Creating secrets for GitLab-sourced apps". Does it all make sense for Konflux?

GitHub isn't mentioned as I was told that only apps created from a GitLab repo will require this special workflow when an access token is added _before_ an app is created. Correct me if this is wrong, please. 
Thanks!

**Comment by missmesss** [comment]
@arewm Can you please rereview? I hope this is a more or less final version.
I got my answers from Max Shaposhnyk's PR. I guess now this guide has everything we hoped to add to it? Thank you.

File: docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc (hunk 2 above)
Comment (mshaposhnik) [reply docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc:28]: LGTM, thanks!

**Review by mshaposhnik — APPROVED** [review]

**Review by ernesgonzalez33 — CHANGES_REQUESTED** [review]

**Review by arewm — APPROVED** [review]
No comments other than those that have already been made.

**Review by ernesgonzalez33 — APPROVED** [review]

**Comment by missmesss** [comment]
@ernesgonzalez33 Thank you for a review! I've committed your suggested change. I guess we don't want a rereview because of `gitlab.com` :) I'll merge now.

**Comment by missmesss** [comment]
@ernesgonzalez33 Ah, I just saw your approval, thank you so much!

**Pull request merged**